src/Th_Bool.ml

```diff
@@ -11,10 +11,11 @@ let new_components = [
     codomain = TBool;
     domain = [TBool];
     check = (function
-             | [FCall ("not", _)] -> false
+             | _ -> true );
+             (* | [FCall ("not", _)] -> false
              | [Const _] -> false
              | [_] -> true
-             | _ -> false);
+             | _ -> false); *)
```

Comment:
Why?!

Reply:
Sorry, forgot to reset debug changes.. Will change back!